packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -381,10 +395,12 @@ class RenderListWheelViewport
     if (value == _renderChildrenOutsideViewport)
       return;
     _renderChildrenOutsideViewport = value;
-    _hasScrolled();
+    markNeedsPaint();
+    markNeedsSemanticsUpdate();
   }
 
   void _hasScrolled() {
+    markNeedsLayout();
     markNeedsPaint();
```

Comment:
you never need to call markNeedsLayout and markNeedsPaint, the former implies the latter